IntentsHandler/IntentHandler.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  IntentHandler.swift
+//  IntentsHandler
+//
+//  Created by Yana Sang on 10/17/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import Intents
+
+// As an example, this class is set up to handle Message intents.
+// You will want to replace this or add other intents as appropriate.
+// The intents you wish to handle must be declared in the extension's Info.plist.
+
+// You can test your example integration by saying things to Siri like:
+// "Send a message using <myApp>"
+// "<myApp> John saying hello"
+// "Search for messages in <myApp>"
+
+class IntentHandler: INExtension, INSendMessageIntentHandling, INSearchForMessagesIntentHandling, INSetMessageAttributeIntentHandling {
```

Comment:
would prefer to have some comments above each method to explain what it does. ik there are some comments already that explain generally what the method does but if you could make some comments that are specific to your implementation of this class it would be helpful.